docs-compose/overwatch_docs/content/FAQ/_index.md

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
This file / entire directory `ovewerwatch_docs` should not be begin committed to main. Please remove this directory from your branch. remember when you switch branches, if the file is open it can't remove it.